Delicate-Butterflies/Bangazon-Site
# Sb user model

Related Issue: #28

Description: User registration and login closer reflect ERD instead of boilerplate. Changed config to properly .gitignore.

IMPORTANT NOTE - WHEN REGISTRATION FORM IS UPDATED TO INCLUDE ADDRESS, PHONE NUMBER, ETC passport-strat.js WILL NEED TO BE UPDATED TO REFLECT THAT.

Chai/Mocha Tests File:

PR Testing Steps:
- fetch branch
- ensure db is up to date on migrations
- run npm:start
- register a user (welcome banner should display username)
- logout then log back in as that user (welcome banner should display username)

Check list before you make a PR:
